R/shared-methods.R

```diff
@@ -97,10 +97,14 @@ initialise_dataclass <- function(self, level = "1", regions,
   }
 
   if (!is.null(self$level_data_urls[[self$level]])) {
-    self$data_urls <- merge(
-      self$common_data_urls,
-      self$level_data_urls[[self$level]]
-    )
+    if (length(self$common_data_urls) > 0) {
+      self$data_urls <- merge(
+        self$common_data_urls,
```

Comment:
I think we want this the other way around so that the default is to use level specific items?

Reply:
My sense - and I've not done a survey - is that most countries have a subset of urls (often just one) which is shared among all levels. Some also have one or two level-specific countries. One or two countries (e.g. USA, France) have no common urls between levels, but I *think* it's better that the "default" be working from the `common_data_urls`.